runtime/complex_test.go

```diff
@@ -508,7 +508,8 @@ func TestComplexHash(t *testing.T) {
 }
 
 func floatsAreSame(a, b float64) bool {
-	return a == b || (math.IsNaN(a) && math.IsNaN(b))
+	EPSILON := 0.00000001
+	return (a == b) || (b-a < EPSILON) || (a-b < EPSILON) || (math.IsNaN(a) && math.IsNaN(b))
```

Comment:
I believe this change is letting some incorrect cases through: the two Div failures are because Go 1.9 moved to C99 float division and that’s what  CPython uses as well. So the expected values should change be updated to the calculated values. This will break Go versions < 1.9 so we probably need some special case code to fix that bug.

The Pow issues are different and I think that CPython does some special casing for whole number inputs to make sure the output is also a whole number. I haven’t looked into that yet though. I’m not sure what’s up with the 3.1415 case.

In any case I don’t think we should do the epsilon diff because it’s not consistent with python float equality.